rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -230,7 +230,6 @@ components:
       required: true
       schema:
         type: string
-        format: uri
```

Comment:
❗️ Although this API is not really out yet, this change is removing a restriction on the attribute that was there in the previous version. We shouldn't be changing the spec to fix test cases. If the new generator is generating the code as a URI now as opposed to a string as before then this is what it should be.

Reply:
I'm not certain that I follow the logic.

If the API originally was generating "String someMethod()" and is now generating "URI someMethod()"; wouldn't it be more consistent in the API to require String to keep backwards compatibility.  Or, are you saying that you prefer the harder restriction of URI and because this API is not released; the signature change won't matter (thereby change the unit test)?

Reply:
Currently what matters is the open api spec and not how the code was generated. The fact that the generator didn't put the right restrictions based on how it was specified is not an indication of what the specification of that version was. I think we need to stick with URI and be as specific as we can be. Code generation is something that has to be fixed separately. In other words, one couldn't say that the spec was accepting any strings since the spec was saying it should a URI. The code might have been wrong bu that is not the spec.

Reply:
Ok, I followed that.  I'll make the necessary changes and get back to you once the unit tests are passing.  Thanks for the clarification.